lib/cosmos/packet_logs/packet_log_writer.rb

```diff
@@ -356,7 +337,9 @@ def get_packet_index(cmd_or_tlm, target_name, packet_name)
 
       # New target_table entry needed
       packet_index = @next_packet_index
-      raise "Packet Table Overflow" if packet_index > COSMOS5_MAX_PACKET_INDEX
+      if packet_index > COSMOS5_MAX_PACKET_INDEX
+        raise "Packet Index Overflow"
+      end
```

Comment:
When we write inline if statements it makes it difficult to determine code which hasn't been run (code-coverage). Consider being a little more verbose with if statements to enable code coverage, especially in cases where we raise errors or break loops like above.